# In degree and meter
H1_Pro:
  Shoulder_Y_R:
    Upper: 64
    Lower: -184
    Startup: 0
  Shoulder_X_R:
    Upper: 0
    Lower: -172
    Startup: -15
  Shoulder_Z_R:
    Upper: 161
    Lower: -161
    Startup: 0
  Elbow_R:
    Upper: 2
    Lower: -112
    Startup: -1
  Wrist_Z_R:
    Upper: 162
    Lower: -162
    Startup: 0
  Wrist_Y_R:
    Upper: 84
    Lower: -84
    Startup: 0
  Wrist_X_R:
    Upper: 84
    Lower: -84
    Startup: 0
  Shoulder_Y_L:
    Upper: 64
    Lower: -184
    Startup: 0
  Shoulder_X_L:
    Upper: 172
    Lower: 0
    Startup: 15
  Shoulder_Z_L:
    Upper: 161
    Lower: -161
    Startup: 0
  Elbow_L:
    Upper: 2
    Lower: -112
    Startup: -1
  Wrist_Z_L:
    Upper: 162
    Lower: -162
    Startup: 0
  Wrist_Y_L:
    Upper: 84
    Lower: -84
    Startup: 0
  Wrist_X_L:
    Upper: 84
    Lower: -84
    Startup: 0
  Neck_Z:
    Upper: 40
    Lower: -40
    Startup: 0
  Neck_Y:
    Upper: 15
    Lower: -5
    Startup: 0
  A_Waist:
    Upper: 85
    Lower: -85
    Startup: 0
WA1_0303:
  Shoulder_Y_R:
    Upper: 64
    Lower: -184
    Startup: 0
  Shoulder_X_R:
    Upper: 0
    Lower: -172
    Startup: -35
  Shoulder_Z_R:
    Upper: 161
    Lower: -161
    Startup: -30
  Elbow_R:
    Upper: 2
    Lower: -112
    Startup: -1
  Wrist_Z_R:
    Upper: 162
    Lower: -162
    Startup: 0
  Wrist_Y_R:
    Upper: 84
    Lower: -84
    Startup: 0
  Wrist_X_R:
    Upper: 84
    Lower: -84
    Startup: 0
  Shoulder_Y_L:
    Upper: 64
    Lower: -184
    Startup: 0
  Shoulder_X_L:
    Upper: 172
    Lower: 0
    Startup: 35
  Shoulder_Z_L:
    Upper: 161
    Lower: -161
    Startup: 30
  Elbow_L:
    Upper: 2
    Lower: -112
    Startup: -1
  Wrist_Z_L:
    Upper: 162
    Lower: -162
    Startup: 0
  Wrist_Y_L:
    Upper: 84
    Lower: -84
    Startup: 0
  Wrist_X_L:
    Upper: 84
    Lower: -84
    Startup: 0
  Neck_Z:
    Upper: 40
    Lower: -40
    Startup: 0
  Neck_Y:
    Upper: 15
    Lower: -5
    Startup: 0
  Waist_Z:
    Upper: 175
    Lower: -111
    Startup: 0
  Waist_Y:
    Upper: 37
    Lower: -22
    Startup: 0
  Lifting_Z:
    Upper: 0.205
    Lower: 0.003
    Startup: 0.01
WA2_A2_lite:
  Shoulder_Z_R:
    Upper: 15
    Lower: -15
    Startup: 0
  Shoulder_Y_R:
    Upper: 64
    Lower: -184
    Startup: 0
  Shoulder_X_R:
    Upper: 0
    Lower: -172
    Startup: -35
  Elbow_Z_R:
    Upper: 161
    Lower: -161
    Startup: -30
  Elbow_Y_R:
    Upper: 2
    Lower: -112
    Startup: -1
  Wrist_Z_R:
    Upper: 162
    Lower: -162
    Startup: 0
  Wrist_Y_R:
    Upper: 84
    Lower: -84
    Startup: 0
  Wrist_X_R:
    Upper: 84
    Lower: -84
    Startup: 0
  Shoulder_Z_L:
    Upper: 15
    Lower: -15
    Startup: 0
  Shoulder_Y_L:
    Upper: 64
    Lower: -184
    Startup: 0
  Shoulder_X_L:
    Upper: 172
    Lower: 0
    Startup: 35
  Elbow_Z_L:
    Upper: 161
    Lower: -161
    Startup: 30
  Elbow_Y_L:
    Upper: 2
    Lower: -112
    Startup: -1
  Wrist_Z_L:
    Upper: 162
    Lower: -162
    Startup: 0
  Wrist_Y_L:
    Upper: 84
    Lower: -84
    Startup: 0
  Wrist_X_L:
    Upper: 84
    Lower: -84
    Startup: 0
  Neck_Z:
    Upper: 40
    Lower: -40
    Startup: 0
  Neck_Y:
    Upper: 15
    Lower: -5
    Startup: 0
  Pitch_Y_B:
    Upper: 55
    Lower: -55
    Startup: 0
  Pitch_Y_M:
    Upper: 135
    Lower: -135
    Startup: 0
  Waist_Z:
    Upper: 180
    Lower: -115
    Startup: 0.01
  Waist_Y:
    Upper: 50
    Lower: -50
    Startup: 0.01
Hand:
  thumb_MP:
    Upper: 57
    Lower: -57
    Startup: 0
  thumb_CMC:
    Upper: 85
    Lower: 0
    Startup: 0
  index_MCP:
    Upper: 85
    Lower: -57
    Startup: 0
  middle_MCP:
    Upper: 85
    Lower: 0
    Startup: 0
  ring_MCP:
    Upper: 85
    Lower: 0
    Startup: 0
  little_MCP:
    Upper: 85
    Lower: 0
    Startup: 0
